benchmarks/asv.conf.json

```diff
@@ -17,7 +17,7 @@
     // uninstalling the project. See asv.conf.json documentation.
     //
     "install_command": [
-        "pip install torch torchvision",
+       // "conda install pytorch torchvision torchaudio",
```

Comment:
remove this line completely. let's focus on the core dependency (numpy) for the time being